gae_discourse_client/content.py

```diff
@@ -0,0 +1,48 @@
+"""Module for accessing and creating content through the Discourse API"""
+
+from google.appengine.ext import ndb
+
+
+class Error(Exception):
+    pass
+
+
+class ContentClient(object):
+    """An API client for interacting with Discourse for content-related actions"""
+
+    def __init__(self, api_client, category_client):
+        self._api_client = api_client
+        self._category_client = category_client
+
+    @ndb.tasklet
+    def get(self, category_id=None, page=0):
+        """Gets all posts for the given category.
```

Comment:
This doesn't seem correct. Aren't we getting a list of topics, not posts, back? Maybe I'm misunderstanding the meaning of the two...